great_expectations/expectations/core/expect_table_row_count_to_be_between.py

```diff
@@ -39,7 +39,10 @@ class ExpectTableRowCountToBeBetween(BatchExpectation):
     expect_table_row_count_to_be_between is a \
     [Batch Expectation](https://docs.greatexpectations.io/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_batch_expectations).
 
-    Keyword Args:
+    BatchExpectations are one of the most common types of Expectation.
+    They are evaluated for an entire Batch, and answer a semantic question about the Batch itself.
```

Comment:
```suggestion
    They evaluate an entire Batch, and answer a semantic question about the Batch.
```
Apply the edit to all similar statements.